test/evergreen.yml

```diff
@@ -6464,6 +5837,137 @@ buildvariants:
     - name: ".stress-test-1"
     - name: ".stress-test-2"
     - name: format-abort-recovery-stress-test
-    - name: ".stress-test-1-nonstandalone"
-    - name: ".stress-test-2-nonstandalone"
-    - name: format-abort-recovery-stress-test-nonstandalone
+
+- name: ubuntu2004-nonstandalone
+  display_name: "Ubuntu 20.04 Nonstandalone"
+  run_on:
+  - ubuntu2004-test
+  expansions:
+    ENABLE_TCMALLOC: -DENABLE_TCMALLOC=1
+    data_validation_stress_test_args: -t r -m -W 3 -D -p -n 100000 -k 100000 -C cache_size=100MB
+    num_jobs: $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
+    NONSTANDALONE: -DWT_STANDALONE_BUILD=0
+    unit_test_hook: --hook nonstandalone
+  tasks:
+    - name: ".pull_request !.pull_request_compilers"
+    - name: compile
+    - name: make-check-test
+    - name: unit-test-extra-long
+      distros: ubuntu2004-large
+    - name: ".data-validation-stress-test"
+  
+- name: ubuntu2004-nonstandalone-stress
+  display_name: "Ubuntu 20.04 Nonstandalone Stress tests"
+  run_on:
+  - ubuntu2004-test
+  expansions:
+    ENABLE_TCMALLOC: -DENABLE_TCMALLOC=1
+    # This isn't normally used in cppsuite
+    additional_env_vars: export LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libeatmydata.so:$LD_PRELOAD"
+    num_jobs: $(echo "`grep -c ^processor /proc/cpuinfo`" | bc)
+    NONSTANDALONE: -DWT_STANDALONE_BUILD=0
+  tasks:
+    - name: ".stress-test-1"
+    - name: ".stress-test-2"
+    - name: ".stress-test-3"
+    - name: ".stress-test-4"
+    - name: ".stress-test-no-barrier"
+    - name: format-abort-recovery-stress-test
+    - name: compile
+    - name: ".cppsuite-stress-test"
+      distros: ubuntu2004-medium
+
+- name: ubuntu2004-nonstandalone-release
+  display_name: "Ubuntu 20.04 Nonstandalone (Release Build)"
+  run_on:
+  - ubuntu2004-small
+  expansions:
+    ENABLE_TCMALLOC: -DENABLE_TCMALLOC=0
```

Comment:
I personally think that we should combine the stress and non-stress variants here if the amount of tests run on a variant is small. For me I see it more code readable, do you think otherwise? If so I am happy to seperate them out.

Reply:
The creation of the variant happened in WT-10642. Unfortunately there is no context on why we didn't add it. I am thinking that we may have just missed adding it and we were not aware of it.